src/elements/content-uploader/progressCellRenderer.js

```diff
@@ -65,6 +71,9 @@ export default () => ({ rowData }: Props) => {
         case STATUS_STAGED:
             return <ItemProgress {...rowData} />;
         case STATUS_ERROR:
+            if (Browser.isSafari() && code === ERROR_CODE_UPLOAD_BAD_DIGEST && file.name.indexOf('.zip') !== -1) {
```

Comment:
Should we destructure out the name on L62?

Reply:
I left this one as a key reference b/c we already had a destructured `name` from rowData in this file.